server/app/lib/actions/fusor/deployment/open_shift/install_image.rb

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+require 'net/scp'
+
+module Actions
+  module Fusor
+    module Deployment
+      module OpenShift
+        class InstallImage < Actions::Fusor::FusorBaseAction
+          def humanized_name
```

Comment:
This file started out as a RPM install for the image.  But we've since changed it to do what it is doing now (tar up and transfer). I agree that I could rename the file to transfer image.